src/components/pages/UpdateFarmer/UpdateFarmer.js

```diff
@@ -119,7 +121,7 @@ const UpdateFarmer = ({ location, history, appStateShouldUpdate }) => {
     window.scrollTo(0, 75);
   };
 
-  const formHandler = e => {
+  const formHandler = async e => {
```

Comment:
We should either use `async await` with `try catch` block or `.then().catch()` syntax. But not both :)

Reply:
Yeah, you're right. Forgot to delete after some experimentation. Good catch! 🔥